websites/O/OpenFING/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"name": "Extra",
+		"id": "351816851380109332"
+	},
+	"service": "OpenFING",
+	"description": {
+		"en": "OpenFING is a student project dedicated to digitizing courses by the UdelaR's Engineering Faculty (Uruguay)",
+		"es": "OpenFING es un proyecto estudiantil dedicado a digitalizar los cursos impartidos en la Facultad de Ingeniería de la UdelaR (Uruguay)."
+	},
+	"url": "open.fing.edu.uy",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/7mO4xCK.png",
+	"thumbnail": "https://i.imgur.com/guoaeqg.png",
+	"color": "#4384fa",
+	"category": "videos",
+	"tags": [
+		"education",
+		"math",
+		"uruguay",
+		"openfing",
+		"engineering"
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*"
```

Comment:
Try to use something else besides `.*`, unless the videos are located in a lot of domains.